### PR TITLE
Changes.

### PR DESCRIPTION
**Pull request by otteric**
- Change old username (`Bax3n`) to real name (or at least part of real name, I'm not putting my last name)
- Added the extension files; with changing domains and stuff, this is a permanent spot
- Changed links to the extension files

### CODE REVIEW

**Comment by savaka2** [comment]
Is it intentional that the "Logic Gates" extension is not listed?

**Comment by c-9** [comment]
I added it now

File: new-extensions.xml [hunk 1]
```diff
@@ -5,6 +5,13 @@
 	<link>http://savaka2.github.io/scratch-extensions-directory/new-extensions.xml</link>
 	<description>New extensions in the Scratch Extensions Directory</description>
 	<item>
+		<title>Transform by Eric O.</title>
```
Comment (savaka2) [inline new-extensions.xml]: What I assume was supposed to be an entry for the new extension "Logic Gates" is just a duplicate of the "Transform" item

use this to generate a date if necessary http://savaka2.github.io/date.html

**Comment by otteric** [comment]
Fixed it.

**Pull request merged**